From 2760bd4ccde8c8ef38a3ca5da82fd8b0249b72a9 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 20 Jun 2014 11:52:22 -0400
Subject: [redhat] RHMAINTAINERS: update RHMAINTAINERS for networking

Message-id: <7590cfc0c85ee943031d7be2a36cd653b53bb080.1403265130.git.jbenc@redhat.com>
Patchwork-id: 84296
O-Subject: [RHEL7.1 PATCH] redhat: update RHMAINTAINERS for networking
Bugzilla:
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

This updates RHMAINTAINERS to the current state, as some people listed there
left the company. There's also an attempt to better specify which files and
directories fall under the networking entries.

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index 066859c..18c4ecd 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -87,6 +87,11 @@ F: include/linux/acpi.h
 F: include/acpi/
 F: arch/x86/kernel/acpi/
 
+AF_PACKET
+M: Daniel Borkmann <dborkman@redhat.com>
+S: Maintained
+F: net/packet/
+
 AIO
 M: Jeff Moyer <jmoyer@redhat.com>
 S: Maintained
@@ -321,13 +326,21 @@ IPMI SUBSYSTEM
 M: Tony Camuso <tcamuso@redhat.com>
 S: Maintained
 
-IPV4/IPV6 PROTOCOLS
+IPV4 PROTOCOL
+M: Jiri Benc <jbenc@redhat.com>
 M: Neil Horman <nhorman@redhat.com>
-M: Thomas Graf <tgraf@redhat.com>
 S: Maintained
 F: net/ipv4
+F: include/linux/ip.h
+F: include/net/ip.h
+
+IPV6 PROTOCOL
+M: Hannes Frederic Sowa <hsowa@redhat.com>
+M: Neil Horman <nhorman@redhat.com>
+S: Maintained
 F: net/ipv6
-F: include/net
+F: include/linux/ipv6.h
+F: include/net/ipv6.h
 
 ISMT SMBUS DRIVER
 M: Neil Horman <nhorman@redhat.com>
@@ -386,22 +399,57 @@ S: Maintained
 F: drivers/net/myri10ge/
 
 NETLINK
-M: Thomas Graf <tgraf@redhat.com>
+M: Daniel Borkmann <dborkman@redhat.com>
 S: Maintained
 F: net/netlink/
-F: include/linux/netlink.h
-F: include/net/netlink.h
+F: include/linux/*netlink*
+F: include/net/*netlink*
 
 NETWORKING [GENERAL]
+M: Jiri Benc <jbenc@redhat.com>
+M: Daniel Borkmann <dborkman@redhat.com>
+M: Rashid Khan <rkhan@redhat.com>
 M: Andy Gospodarek <agospoda@redhat.com>
 M: Neil Horman <nhorman@redhat.com>
-M: Thomas Graf <tgraf@redhat.com>
 S: Maintained
 F: net/
+F: Documentation/networking/
+F: drivers/net/loopback.c
+F: drivers/net/Space.c
 F: include/net/
-F: include/linux/in.h
-F: include/linux/net.h
-F: include/linux/netdevice.h
+F: include/linux/eth*
+F: include/linux/filter.h
+F: include/linux/icmp*
+F: include/linux/if*
+F: include/linux/in[.6e]*
+F: include/linux/ip.h
+F: include/linux/ipv6*
+F: include/linux/llc.h
+F: include/linux/mroute*
+F: include/linux/net*
+F: include/linux/skbuff.h
+F: include/linux/sock*
+F: include/linux/tc_act/tc_defact.h
+F: include/linux/tcp.h
+F: include/linux/udp.h
+F: tools/net/
+X: include/net/*80211*
+X: include/net/*80215*
+X: include/net/bluetooth/
+X: include/net/irda/
+X: include/net/iucv/
+X: include/net/iw*
+X: include/net/regulatory.h
+X: net/bluetooth/
+X: net/ieee802154/
+X: net/irda/
+X: net/iucv/
+X: net/mac80211/
+X: net/rds/
+X: net/rfkill/
+X: net/sunrpc/
+X: net/wimax/
+X: net/wireless/
 
 NETXEN (1/10) GbE SUPPORT
 M: Chad Dupuis <cdupuis@redhat.com>
@@ -449,8 +497,6 @@ F: include/linux/nvme.h
 F: include/uapi/linux/nvme.h
 
 OPENVSWITCH (OVS)
-M: Thomas Graf <tgraf@redhat.com>
-M: Francesco Fusco <ffusco@redhat.com>
 M: Flavio Leitner <fbl@redhat.com>
 S: Maintained
 F: net/openvswitch/
@@ -536,10 +582,11 @@ S: Maintained
 F: drivers/scsi/
 
 SCTP PROTOCOL
+M: Daniel Borkmann <dborkman@redhat.com>
 M: Neil Horman <nhorman@redhat.com>
-M: Thomas Graf <tgraf@redhat.com>
 S: Maintained
 F: net/sctp/
+F: include/linux/sctp.h
 F: include/net/sctp/
 
 SERIAL ATA (SATA) SUBSYSTEM
-- 
1.7.1